tests/test_onnxml_sv_converter.py

```diff
@@ -0,0 +1,112 @@
+"""
+Tests onnxml SV converters
+"""
+import unittest
+import warnings
+
+import numpy as np
+import torch
+from sklearn.svm import LinearSVC, SVC, NuSVC
+
+from hummingbird.ml._utils import onnx_ml_tools_installed, onnx_runtime_installed, lightgbm_installed
+from hummingbird.ml import convert
+
+if onnx_runtime_installed():
+    import onnxruntime as ort
+if onnx_ml_tools_installed():
+    from onnxmltools import convert_sklearn
+    from onnxmltools.convert.common.data_types import FloatTensorType as FloatTensorType_onnx
+
+
+class TestONNXSVC(unittest.TestCase):
+    def _test_sv(self, classes, mode="torch"):
+        """
+        This helper function tests conversion of `ai.onnx.ml.SVMClassifier`
+        which is created from a scikit-learn SVC.
+
+        This then calls either "_to_onnx" or "_to_torch"
+        """
+        n_features = 20
+        n_total = 100
+        np.random.seed(0)
+        warnings.filterwarnings("ignore")
+        X = np.random.rand(n_total, n_features)
+        X = np.array(X, dtype=np.float32)
+        y = np.random.randint(classes, size=n_total)
+
+        # Create SKL model for testing
+        model = SVC()
+        model.fit(X, y)
+
+        # Create ONNX-ML model
+        onnx_ml_model = convert_sklearn(model, initial_types=[("float_input", FloatTensorType_onnx(X.shape))])
+
+        # Get the predictions for the ONNX-ML model
+        session = ort.InferenceSession(onnx_ml_model.SerializeToString())
+        output_names = [session.get_outputs()[i].name for i in range(len(session.get_outputs()))]
+        onnx_ml_pred = [[] for i in range(len(output_names))]
+        inputs = {session.get_inputs()[0].name: X}
+        pred = session.run(output_names, inputs)
+        for i in range(len(output_names)):
+            if "label" in output_names[i]:
+                onnx_ml_pred[1] = pred[i]
+            else:
+                onnx_ml_pred[0] = pred[i]
+
+        if mode == "torch":
```

Comment:
I don't think we need this if, with the containers now we do the same thing between onnx and torch models.

Reply:
I can pull out `convert` and pass the mode as an arg, but lines 58 (`pred = torch_model.predict(X)`) and `pred = [[] for i in range(len(output_names))]`...etc are different, unless you see a clean way to combine them?

Reply:
oh wait, you're right, we don't need that. let me fix